### DISCUSSION TITLE
tf.image.decode_image can not output image shape

### DISCUSSION THREAD

**Discussion by Yaoxingtian** [tensorflow/compression#92] [General]
hi, thanks for your opensource. the problem is as below,  tf1.15 version cannot work of tf.image.decode_image, any solutions to solve this problem ? thanks so much!

File "tfci.py", line 124, in compress
    bitstring = compress_image(model, input_image)
  File "tfci.py", line 107, in compress_image
    packed.pack(tensors)
TypeError: pack() missing 1 required positional argument: 'arrays'

**Comment by jonaballe** [comment]
Hi, this is most likely a mismatch between versions. If you are using TF 1.15 and TFC 1.3, you need to download the model script from [here](https://github.com/tensorflow/compression/tree/e1e08a2c62e4d08b93c6bf4008c8a123fc17b2a0/examples). This is the corresponding older version of the script.
